README.md

```diff
@@ -86,6 +86,20 @@ $databases['default']['default'] = array (
 );
 ```
 
+### Memcached settings
```

Comment:
Should read "Memcache settings", since "memcached" is the reserver-side daemon, while "memcache" is the PHP extension used to talk to it. The same applies to all instances below.